update_docker_image.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+# DEVELOPER: update this field anytime you make a new docker image
+VERSION="2.1"
+BUILD_TAG="broadinstitute/horsefish:v${VERSION}"
+
+echo "using build tag: ${BUILD_TAG}"
+
+if docker manifest inspect $BUILD_TAG > /dev/null ;
+then
+    echo "Docker image already exists. Do you want to overwrite this tag?."
```

Comment:
would be nice to print the name of the tag here (I know you just echoed it above - but still) - could change this to 
`echo "Docker image with tag ${BUILD_TAG} already exists. Do you want to overwrite this tag?"`

Reply:
HA - I hadn't even read this yet and added that in. Great minds!

Reply:
did you add it in? i don't see it. not a huge deal :shrug: